notebooks/amazon_forest_notebook.py

```diff
@@ -208,7 +209,7 @@
 
 # <codecell>
 
-from keras.callbacks import ModelCheckpoint
+from tensorflow.contrib.keras.api.keras.callbacks import ModelCheckpoint
 
 filepath="weights.best.hdf5"
 checkpoint = ModelCheckpoint(filepath, monitor='val_acc', verbose=1, save_best_only=True, mode='max')
```

Comment:
Yes the output isn't pretty but its nice to see at a quick glance that the model has achieved a new best weight on a particular epoch. Otherwise you have to search for the accuracy numbers to see for yourself which one is the best.